rust/processor/src/db/common/models/fungible_asset_models/v2_fungible_asset_balances.rs

```diff
@@ -165,6 +165,12 @@ impl FungibleAssetBalance {
                 let asset_type = inner.metadata.get_reference_address();
                 let is_primary = Self::is_primary(&owner_address, &asset_type, &storage_id);
 
+                // If store has the untransferable field then it must be frozen
+                let is_frozen = if object_data.untransferable.as_ref().is_some() {
+                    true
+                } else {
+                    inner.frozen
+                };
```

Comment:
frozen and untransferable are 2 different concepts for fungible assets. we should split them out into separate columns

Reply:
questions: how are they different? can we transfer frozen ones?

Reply:
I think you need to undo this change.